contrib/cronie/template.py

```diff
@@ -0,0 +1,43 @@
+pkgname = "cronie"
+pkgver = "1.7.1"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--enable-anacron",
+    "--with-inotify",
+    "--with-pam",
+    "--without-selinux",
+]
+make_cmd = "gmake"
+hostmakedepends = ["gmake", "autoconf", "automake"]
+makedepends = ["linux-pam-devel"]
+depends = ["bash", "debianutils"]
+pkgdesc = "Cronie cron daemon project"
+maintainer = "flukey <flukey@vapourmail.eu>"
+# TODO: obstack in cports causes LTO build errors, bundled obstack can be used
+# instead, but its license is LGPL-2.1, which cbuild apparently doesnt like?
```

Comment:
because the actual licence is either lgpl-2.1-only or lgpl-2.1-or-later

what is the error though?

Reply:
```
ld: error: undefined symbol: _obstack_begin
>>> referenced by readtab.c:400 (./build/../anacron/readtab.c:400)
```

this is because it's never linked. you need to add -lobstack somewhere

Reply:
The error was: `ERROR: failed validating license: unknown token: LGPL-2.1`

I found 'LGPL-2.1' on the SPDX license list here: https://spdx.org/licenses/archive/archived_ll_v2.4/LGPL-2.1.html

But I've now added `-lobstack` to LDFLAGS, so this is no longer needed. Thanks!

Reply:
> The error was: ERROR: failed validating license: unknown token: LGPL-2.1

yes, as i said it's an incomplete token

> I found 'LGPL-2.1' on the SPDX license list here:

the url has `archived` in it :) see https://spdx.org/licenses/LGPL-2.1.html for what it "used to mean"